SimPEG/Regularization.py

```diff
@@ -1280,15 +1280,30 @@ def __init__(self, mesh, **kwargs):
     model = properties.Array(
         "current model", dtype=float
     )
+
     gamma = properties.Float(
         "Model norm scaling to smooth out convergence", default=1.
     )
+
     epsilon = properties.Float(
-        "Threshold value for the model norm",  #, default=1e-1
+        "Threshold value for the model norm", default=1e-3,
         required=True
     )
-    norm = properties.Float(
-        "norm used", default=2
+
+    norm = properties.Array(
+        "norm used", dtype=float
+    )
+
+    space = properties.String(
+        "By default inherit the objctive", default='linear'
```

Comment:
I am not really clear on what this is? If it is internal only (eg. the user should not set it), could you make it private? 
`_space`